DUED.html

```diff
@@ -612,15 +612,49 @@
       };
 
 
+      var appendSelectNodes = function (n) {
 
+          var select;
+          var t1 = performance.now();
+          for (var i = 0; i < n; i++) {
+              select = document.createElement('select');
```

Comment:
Adding lots of select elements to a div isn't much different from the other append test. The select element element should be the container to which you add <option> or other elements.